sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -591,6 +594,8 @@ export class ApplicationOfferingChangeRequestService {
         applicationOfferingChangeRequestStatus ===
         ApplicationOfferingChangeRequestStatus.Approved
       ) {
+        // Access the student assessment if it exists.
```

Comment:
The student assessment will be always present in this scenario. It will not be present only when the application is in draft.